File: main/src/main/java/com/example/main/data/source/remote/ApiUtilsMain.java
package com.example.main.data.source.remote;

import com.example.core.BuildConfig;
import com.example.core.data.source.remote.CallApi;

public class ApiUtilsMain {

    public WebServiceMain getListMovieService() {
        return CallApi.getClient(BuildConfig.BASE_URL).create(WebServiceMain.class);
    }
}

File: main/src/main/java/com/example/main/data/source/remote/ApiUtilsMain.kt
package com.example.main.data.source.remote

import com.example.core.BuildConfig
import com.example.core.data.source.remote.CallApi

class ApiUtilsMain {

    val listMovieService: WebServiceMain
        get() = CallApi.getClient(BuildConfig.BASE_URL).create(WebServiceMain::class.java)
}
